topic-operator/src/test/java/io/strimzi/operator/topic/TopicControllerIT.java

```diff
@@ -102,40 +101,31 @@
 @ExtendWith(KafkaClusterExtension.class)
```

Comment:
Since the `BatchingTopicControllerTest` comes with a comment that points to this test, let's add a comment here pointing to that one and similarly explain how you'd decide where a new test should go.

Reply:
Ok. What do you think about using MockKube in BatchingTopicControllerTest and other unit tests. Maybe also something similar for Kafka? I will explore that after this refactoring series is complete.